src/codegen_expr.cpp

```
@@ -1,7 +1,16 @@
 #include "ry/codegen.hpp"
+#include "ry/stdlib_registry.hpp"
 #include "ry/diagnostic.hpp"
 #include <climits>
 
+static int rk_regex;
```

Clang (14.0.6)

[warning] variable 'rk_regex' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)